x11/wine-crossover/Portfile

```diff
@@ -118,26 +108,49 @@ configure.args              --without-alsa \
                             --without-ldap \
                             --without-mpg123 \
                             --without-openal \
-                            --with-opengl \
                             --without-oss \
                             --with-png \
                             --with-pthread \
                             --without-sane \
                             --with-tiff \
                             --without-v4l \
-                            --with-xcomposite \
-                            --with-xcursor \
-                            --with-xinerama \
-                            --with-xinput \
+                            --without-x \
                             --with-xml \
-                            --with-xrandr \
-                            --with-xrender \
-                            --with-xshape \
-                            --with-xslt \
-                            --with-xxf86vm \
-                            --with-x \
-                            --x-include=${prefix}/include \
-                            --x-lib=${prefix}/lib
+                            --with-xslt
+
+# Wine has a native macOS driver since 1.5.26 so X11 is optional
+variant x11 {
+  depends_lib-append          port:libGLU \
+                              port:mesa \
+                              port:xorg-libX11 \
+                              port:xorg-libXcomposite \
+                              port:xorg-libXcursor \
+                              port:xorg-libXext \
+                              port:xorg-libXi \
+                              port:xorg-libXinerama \
+                              port:xorg-libXrandr \
+                              port:xorg-libXxf86vm \
+                              port:xorg-libsm \
+                              port:xorg-xf86vidmodeproto \
+                              port:xrender
+
+  configure.args-delete       --without-x
+
+  configure.args-append       --with-opengl \
+                              --with-xcomposite \
+                              --with-xcursor \
+                              --with-xinerama \
+                              --with-xinput \
+                              --with-xrandr \
+                              --with-xrender \
+                              --with-xshape \
+                              --with-xxf86vm \
+                              --with-x \
+                              --x-include=${prefix}/include \
+                              --x-lib=${prefix}/lib
+}
+
+default_variants            +x11
```

Comment:
I thought it could still be useful to have it be built by the buildbot. The MacDriver is still the preferred way that it launches so you would have to explicitly ask for the X11 one to load. Of course, if you're building from scratch it is nice to not have to pull in all the X11 dependencies.

Not sure if `+quartz` would be a necessity, at least, not now. Maybe if in the future macOS gets a third output option then perhaps?